recipes/metabat2/meta.yaml

```
@@ -21,6 +23,8 @@ requirements:
     - {{ compiler('c') }}
     - cmake
     - autoconf
+    - automake
+    - libtool
     - llvm-openmp  # [osx]
     - libgomp      # [linux]
```

```suggestion
    - libgomp      # [linux]
    - sysroot_linux-aarch64 >= 2.28 # [linux and aarch64]
```
https://github.com/samtools/htscodecs/issues/88#issuecomment-1664421461